packages/docs/src/styles/home.css

```
@@ -1,590 +1,1631 @@
+
+.spec-item code {
+  max-width: 100%;
+  overflow: hidden;
+  padding: 0;
+  border: 0;
   color: var(--text-primary);
-  margin-bottom: 0.25rem;
-  font-weight: 600;
-  line-height: 1.2;
+  background: transparent;
+  font-family: 'Consolas', 'Monaco', 'Courier New', monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)